sys/linux/dev_loop.txt

```diff
@@ -22,12 +23,19 @@ resource fd_loop_ctrl[fd]
 resource fd_loop_num[intptr]: 0, 1, 2, 10, 11, 12
 openat$loop_ctrl(fd const[AT_FDCWD], file ptr[in, string["/dev/loop-control"]], flags flags[open_flags], mode const[0]) fd_loop_ctrl
 ioctl$LOOP_CTL_GET_FREE(fd fd_loop_ctrl, cmd const[LOOP_CTL_GET_FREE]) fd_loop_num
-ioctl$LOOP_CTL_ADD(fd fd_loop_ctrl, cmd const[LOOP_CTL_ADD], num fd_loop_num)
+ioctl$LOOP_CTL_ADD(fd fd_loop_ctrl, cmd const[LOOP_CTL_ADD], num fd_loop_num) fd_loop
```

Comment:
It does not seem to return a loop fd.
It returns `i`:
https://elixir.bootlin.com/linux/v5.18-rc5/source/drivers/block/loop.c#L2073
which is assigned here:
https://elixir.bootlin.com/linux/v5.18-rc5/source/drivers/block/loop.c#L1997
which comes from idr_alloc for loop indices:
https://elixir.bootlin.com/linux/v5.18-rc5/source/drivers/block/loop.c#L1988
So it seems to be a loop index rather than a file descriptor.
If you agree, please revert this change.